exporter/fileexporter/file_exporter_test.go

```diff
@@ -132,3 +127,17 @@ func tempFileName(t *testing.T) string {
 	require.NoError(t, os.Remove(socket))
 	return socket
 }
+
+// ErrorWriter is an io.Writer that will return an error all ways
+type ErrorWriter struct {
```

Comment:
s/ErrorWriter/errWriter

No need to be public.

Reply:
Resolved